src/state/action-creators.js

```diff
@@ -450,7 +452,7 @@ export const lift = (id: DraggableId,
   setTimeout(() => {
     const state: State = getState();
 
-    if (state.phase !== 'IDLE' || state.phase !== 'DRAG_COMPLETE') {
+    if (state.phase !== 'IDLE' && state.phase !== 'DRAG_COMPLETE') {
```

Comment:
changed this because I assume this was the intended functionality? the original statement will always evaluate to true.